lib/ProductOpener/Display.pm

```diff
@@ -7044,7 +7054,7 @@ sub display_page ($request_ref) {
 	my $type;
 	my $id;
 
-	my $site = "<a href=\"/\">" . lang("site_name") . "</a>";
+	my $site = "<a href=\"/\">" . $options{site_name} . "</a>";
```

Comment:
why not:
```suggestion
	my $site = "<a href=\"/\">" . lang($options{site_name}) . "</a>";
```
?

Reply:
I see we drop it also below, so maybe there is a good reason ? Is that because you consider it's not to be translated (Open Food Facts).

Reply:
Right, we should not translate Open Food Facts.